src/tests/var/test06.vtc

```diff
@@ -0,0 +1,35 @@
+varnishtest "Test that running out of workspace memory fails the request"
+
+server s1 {
+	rxreq
+	txresp
+} -start
+
+# This bit was tricky to get right:
+#
+# * we are limiting the workspace size to 9k (empirically determined minimum),
```

Comment:
it's not tricky anymore :-)
you can also remove `-arg "-p workspace_backend=9k -p workspace_client=9k"` from the command line since we don't need it anymore